docs/products/postgresql/howto/pagila.rst

```diff
@@ -0,0 +1,263 @@
+Pagila – a PostgreSQL sample database
+=====================================
+
+Aiven provides a sample database you can import in your Aiven for PostgreSQL service. This page covers information about the database and the procedure to get it up and running.
+
+Pagila is a port of the `Sakila Sample Database <https://dev.mysql.com/doc/sakila/en/>`_. It was originally developed by Mark Hillyer, a former member of the MySQL AB documentation team, to provide a standard schema and data for articles, tutorials, books etc. Over time, it has been ported by the various developers to `multiple database systems <https://github.com/search?q=sakila>`_. The examples here use one from ``devrimgunduz``, `version 2.1.0 <https://github.com/devrimgunduz/pagila>`_.
+
+Sakila (and thus, Pagila) is a database representing a DVD rental store (remember those?!), containing information about films (like title, category, actresses), rental stores (like address, staff members, customers) and rentals, where a customer rents a film from a store through its staff.
+
+With all these relational information, Pagila is a perfect fit to play around with PostgreSQL and the SQL language.
+
+Load Pagila on PostgreSQL
+-------------------------
+
+Before exploring the Pagila database, follow the :doc:`create new service article<../../../platform/howto/create_new_service>` to spin up a PostgreSQL instance.
+
+With your new PostgreSQL service, you can quickly load the sample dataset by clicking on the "Load sample dataset" button.
+
+.. image:: /images/products/postgresql/pagila-load-sample-dataset.png
+    :alt: A screenshot of the Aiven Console showing the "Load sample dataset" button.
+
+After loading the data, connect to the PostgreSQL instance using the command below. The ``SERVICE_URI`` value can be found in the Aiven Console dashboard.
+
+.. code:: shell
+
+   psql 'SERVICE_URI'
+
+You can also load it manually by following the steps below:
+
+1. Download the ``pagila-data.sql`` from our `GitHub repository <https://github.com/aiven/devportal/blob/paglia-sample-database/code/products/postgresql/pagila/pagila-data.sql>`_.
```

Comment:
As soon as we merge, I'll push a new PR to fix this link. I did like this so people could test the whole article and because the CI was complaining about missing links heh